README.md

```diff
@@ -1,4 +1,69 @@
-# nc-ext-caldav-carddav-push
-Nextcloud extension to support WebDAV-Push on calendars/address books
+![Development stage: alpha](https://img.shields.io/badge/development%20stage-preview-blue)
 
-This repo is intended as a sample implementation of [https://github.com/bitfireAT/webdav-push/](WebDAV-Push) for Nextcloud calendars and address books.
+_Note:_ If you're coming from the Nextcloud Conf 2024 lightning talk, you can find the [slides here](https://cloud.bitfire.at/s/WoPZaEnC9tmNxa6).
```

Comment:
We can use Github's own [notes function](https://github.com/orgs/community/discussions/16925), I think they look quite good:

```markdown
> [!NOTE]  
> If you're coming from the Nextcloud Conf 2024 lightning talk, you can find the [slides here](https://cloud.bitfire.at/s/WoPZaEnC9tmNxa6).
```

> [!NOTE]  
> If you're coming from the Nextcloud Conf 2024 lightning talk, you can find the [slides here](https://cloud.bitfire.at/s/WoPZaEnC9tmNxa6).